src/components/Filters/ViewsMenu.js

```diff
@@ -89,33 +83,27 @@ class ViewsMenu extends Component {
       views.reduce((sum, item) => sum + item.data.length, 0) > 0
     return (
       <Wrapper>
-        <Button
-          onClick={() =>
-            this.setState({ showViewsMenu: !this.state.showViewsMenu })}
+        <DropDownButton
+          joined
+          alignRight
+          noListFormat
+          label={
+            <span>
+              <FaPieChart style={{ marginRight: 10 }} />
+              Views
+            </span>
+          }
         >
-          <FaPieChart style={{ marginRight: 10 }} />
-          Views
-          <FaCaretDown style={{ float: 'right' }} name='caret-down' />
-        </Button>
-        {this.state.showViewsMenu && (
-          <Fixed
-            z={1}
-            onClick={() => this.setState({ showViewsMenu: false })}
-            top
-            right
-            bottom
-            left
-          />
-        )}
-        {this.state.showViewsMenu && (
-          <MenuPanel className='views-menu__panel'>
+          <Box py={1}>
             {!hasViews && (
-              <Box p={3}>{"You haven't created any views yet"}</Box>
+              <Box py={2} px={3}>
+                {"You haven't created any views yet"}
```

Comment:
Super-NIT: How about backticks? I though that react devs hate double quotes :confused:

Reply:
This is auto-formatted by prettier, but maybe I can nudge it

Reply:
No worries, forget it :+1: